addons/im_livechat/controllers/main.py

```diff
@@ -190,7 +190,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                 'name': channel_vals['name'],
                 'chatbot_current_step_id': channel_vals['chatbot_current_step_id'],
                 'state': 'open',
-                'operator_pid': (operator_partner.id, display_name.replace(',', '')),
+                'operator_pid': (operator_partner.id, display_name.replace(',', ''), fields.Datetime.to_string(operator_partner.write_date)),
```

Comment:
Formatter could be changed to return actual persona data to insert rather than a strange format.